### PR TITLE
Add v4.0.1 release and notes, some incubator removals.

### PR DESCRIPTION
**Pull request by DLotts**

### CODE REVIEW

File: content/_posts/2020-12-22-release-4.0.1.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+categories: [release]
+---
+{% comment %}
+Some of the variables above are used on the summary page that iterates through posts where catagories=release.
+{% endcomment %}{% capture apacheDistMirror %}http://www.apache.org/dyn/closer.lua/rya/{{ page.fullVersion }}{% endcapture %}
+{% capture d %}https://www.apache.org/dist/rya/{{ page.fullVersion }}{% endcapture %}
+
+
+This is Rya's fifth Apache release, version {{ page.version }}.  It is available as a source code download:
+
```
Comment (adinancr) [inline content/_posts/2020-12-22-release-4.0.1.md:19]: We should also say that this is the first release as a top-level project.

File: content/download.md [hunk 2]
```diff
@@ -62,20 +68,20 @@ Release          | Date       | Download | Notes
 <br>
 Download a source distribution in <!-- either *tar* or --> *zip* format,
 and [verify](http://www.apache.org/dyn/closer.cgi#verify)
-using the corresponding *pgp* signature (using the committer file in [KEYS](https://www.apache.org/dist/{{ site.data.project.incubator_slash_name }}/KEYS)).
+using the corresponding *pgp* signature (using the committer file in [KEYS](https://www.apache.org/dist/{{ site.data.project.unix_name }}/KEYS)).
 If you cannot do that, the *sha1* or *sha512* hash file may be used to check that the
 download has completed okay.
```
Comment (adinancr) [inline content/download.md:74]: I think sha1 hash is no longer provided, so just reference the sha512 hash

File: content/download.md [hunk 3]
```diff
@@ -62,20 +68,20 @@ Release          | Date       | Download | Notes
...
 
 
 For fast downloads, current source distributions are hosted on mirror servers;
 
-<!-- older source distributions are in the
-[archive](http://archive.apache.org/dist/{{ site.data.project.incubator_slash_name }}/).
---> 
+Older source distributions are in the
+[archive](http://archive.apache.org/dist/{{ site.data.project.unix_name }}/).
```
Comment (adinancr) [inline content/download.md]: Let's use https:// instead of http://

File: content/download.md [hunk 4]
```diff
@@ -62,20 +68,20 @@ Release          | Date       | Download | Notes
 <br>
 Download a source distribution in <!-- either *tar* or --> *zip* format,
 and [verify](http://www.apache.org/dyn/closer.cgi#verify)
```
Comment (adinancr) [inline content/download.md]: use https://www.apache.org/dyn/closer.cgi#verify

File: content/_posts/2020-12-22-release-4.0.1.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+description: details of the fifth release of Rya
+version: 4.0.1
+fullVersion: rya-4.0.1
+filename: rya-project-4.0.1-source-release
+releaseNotes:  "{% post_url 2020-12-22-relnotes-4.0.1 %}"
+categories: [release]
+---
+{% comment %}
+Some of the variables above are used on the summary page that iterates through posts where catagories=release.
+{% endcomment %}{% capture apacheDistMirror %}http://www.apache.org/dyn/closer.lua/rya/{{ page.fullVersion }}{% endcapture %}
```
Comment (adinancr) [inline content/_posts/2020-12-22-release-4.0.1.md]: use https:// instead of http://

File: content/_posts/2020-12-22-release-4.0.1.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+This is Rya's fifth Apache release, version {{ page.version }}.  It is available as a source code download:
+
+- [Source download from Apache]({{ apacheDistMirror }}/{{ page.filename }}.zip)
+- Verify: {% comment %}
+{% endcomment %} [pgp]( {{ d }}/{{ page.filename }}.zip.asc ){% comment %}
+{% endcomment %} [sha512]({{ d }}/{{ page.filename }}.zip.sha512)
+- [Release notes]({{ page.releaseNotes | liquify }})
+- [Central repository for Maven and other dependency managers](https://search.maven.org/#search%7Cga%7C1%7Cg%3A%22org.apache.rya%22)
+
+[Verify](http://www.apache.org/dyn/closer.cgi#verify)
```
Comment (adinancr) [inline content/_posts/2020-12-22-release-4.0.1.md]: use https:// instead of http://

File: content/_posts/2020-12-22-release-4.0.1.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+using the corresponding *pgp* signature (using the committer file in
+[KEYS](https://www.apache.org/dist/{{ site.data.project.name }}/KEYS)).
+If you cannot do that, the *sha512* hash file may be used to check that the
+download has completed okay.
+
+For fast downloads, current source distributions are hosted on mirror servers;
+older source distributions are in the
+[archive](http://archive.apache.org/dist/{{ site.data.project.unix_name }}/).
```
Comment (adinancr) [inline content/_posts/2020-12-22-release-4.0.1.md]: use https:// instead of http://

File: content/download.md [hunk 8]
```diff
@@ -35,17 +35,23 @@ If you are interested in helping out, please join our developers list [{{site.da
...
+{% endcomment %}{% endif %}{% comment %}
 {% endcomment %}{% if post.fullVersion %}{% comment %}
 {% endcomment %}{% assign v = post.fullVersion %}{% comment %}
 {% endcomment %}{% else %}{% comment %}
 {% endcomment %}{% capture v %}apache-{{ site.data.project.unix_name }}-{{ post.version }}{% endcapture %}{% comment %}
 {% endcomment %}{% endif %}{% comment %}
 {% endcomment %}{% if forloop.index0 < 1 %}{% comment %}
-{% endcomment %}{% capture p %}http://www.apache.org/dyn/closer.lua/{{ site.data.project.incubator_slash_name }}/{{ v }}{% endcapture %}{% comment %}
-{% endcomment %}{% capture d %}https://www.apache.org/dist/{{ site.data.project.incubator_slash_name }}/{{ v }}{% endcapture %}{% comment %}
+{% endcomment %}{% capture p %}http://www.apache.org/dyn/closer.lua/{{ theName }}/{{ v }}{% endcapture %}{% comment %}
```
Comment (adinancr) [inline content/download.md]: use https:// instead of http://

File: content/download.md [hunk 9]
```diff
@@ -35,17 +35,23 @@ If you are interested in helping out, please join our developers list [{{site.da
...
+{% endcomment %}{% capture d %}https://www.apache.org/dist/{{ theName }}/{{ v }}{% endcapture %}{% comment %}
 {% endcomment %}{% else %}{% comment %}
-{% endcomment %}{% capture p %}http://archive.apache.org/dist/incubator/{{ site.data.project.unix_name }}/{{ v }}{% endcapture %}{% comment %}
-{% endcomment %}{% capture d %}https://archive.apache.org/dist/incubator/{{ site.data.project.unix_name}}/{{ v }}{% endcapture %}{% comment %}
+{% endcomment %}{% capture p %}http://archive.apache.org/dist/{{ theName }}/{{ v }}{% endcapture %}{% comment %}
```
Comment (adinancr) [inline content/download.md]: use https:// instead of http://

**Comment by amihalik** [comment]
Looks good!

**Review by adinancr — CHANGES_REQUESTED** [review]
Looks good overall, but I made a few suggestions you should make

**Comment by adinancr** [comment]
Looks great, thanks!

**Pull request merged**